presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -282,6 +282,79 @@ original state.
      comment   | varchar |       |
     (4 rows)
 
+Extra Hidden Metadata Tables
+----------------------------
+
+The Iceberg connector exposes extra hidden metadata tables. You can query these
+as a part of SQL query by appending name with the table.
+
+* ``$properties`` : General properties of the given table
+.. code-block:: sql
+
+    SELECT * FROM "ctas_nation$properties";
+
+.. code-block:: text
+
+             key           |  value
+     ----------------------+---------
+      write.format.default | PARQUET
+
+* ``$history`` : History of table state changes
+.. code-block:: sql
+
+    SELECT * FROM "ctas_nation$history";
+
+.. code-block:: text
+
+               made_current_at            |     snapshot_id     | parent_id | is_current_ancestor
+    --------------------------------------+---------------------+-----------+---------------------
+    2022-11-25 20:56:31.784 Asia/Kolkata  | 7606232158543069775 | NULL      | true
+
+* ``$snapshots`` : Details about the table snapshots
+.. code-block:: sql
+
+    SELECT * FROM "ctas_nation$snapshots";
+
+.. code-block:: text
+
+                 committed_at             |     snapshot_id     | parent_id | operation |                                                  manifest_list                                           |                                                                                 summary
+    --------------------------------------+---------------------+-----------+-----------+----------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+    2022-11-25 20:56:31.784 Asia/Kolkata | 7606232158543069775 | NULL      | append     | s3://my-bucket/ctas_nation/metadata/snap-7606232158543069775-1-395a2cad-b244-409b-b030-cc44949e5a4e.avro | {changed-partition-count=1, added-data-files=1, total-equality-deletes=0, added-records=25, total-position-deletes=0, added-files-size=1648, total-delete-files=0, total-files-size=1648, total-records=25, total-data-files=1}
```

Comment:
![Screenshot 2023-06-12 at 10 01 26 AM](https://github.com/prestodb/presto/assets/7980573/ed2a3101-6b18-4938-bd72-1ec588f1786f)

Looks like formatting is slightly off

Reply:
Corrected. And added spec documentation for snapshots & manifests